### PR TITLE
Chore: prune obsolete hooks

### PR DESCRIPTION
**Pull request by scottrippey**
### Description

Removes unused files.

#### Type of Change

- [x] Refactoring (non-breaking code improvement, cleanup, organization, or optimization)

### How Has This Been Tested?

Example slides have been regression tested.  Touch and keyboard navigation has been validated.

### Checklist: (Feel free to delete this section upon completion)

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have updated type definitions in `index.d.ts` for any breaking API changes
- [x] My code follows the style guidelines of this project (I have run `yarn format`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes (I have run `yarn test`)
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by scottrippey** [comment]
Noteworthy changes; I validated that references are gone to all these files.
Additionally:
`use-keyboard-controls` is no longer used; we use `mousetrap` now
`use-touch-controls` is no longer used; we use `react-swipeable` now

**Comment by ryan-roemer** [comment]
I believe that there are now unused production dependencies from the three factor. Can we scour generally for unused dependencies?

File: package.json [hunk 1]
```diff
@@ -68,8 +66,7 @@
     "ulid": "^2.3.0",
     "unified": "^9.0.0",
     "unist-util-visit": "^2.0.3",
-    "use-resize-observer": "^6.1.0",
-    "wonka": "^3.2.0"
```
Comment (scottrippey) [inline package.json:72]: Wonka was being used in `use-transition-pipe.js`, which was deleted.

File: package.json [hunk 2]
```diff
@@ -45,13 +45,11 @@
     "broadcastchannel-polyfill": "^1.0.0",
     "dedent": "^0.7.0",
     "history": "^4.9.0",
-    "marksy": "^8.0.0",
     "mdast-builder": "^1.1.1",
     "mdast-util-assert": "^3.0.0",
     "mdast-zone": "^4.0.0",
     "merge-anything": "^3.0.3",
     "mousetrap": "^1.6.5",
-    "normalize-newline": "^3.0.0",
```
Comment (scottrippey) [inline package.json:54]: Did not find any usages of this lib in the codebase.  Does it seem safe to remove?  (Removing dependencies is hard without full context!)

File: package.json [hunk 3]
```diff
@@ -45,13 +45,11 @@
     "broadcastchannel-polyfill": "^1.0.0",
     "dedent": "^0.7.0",
     "history": "^4.9.0",
-    "marksy": "^8.0.0",
```
Comment (scottrippey) [inline package.json:48]: Did not find any usages of this lib in the codebase.  Does it seem safe to remove?  (Removing dependencies is hard without full context!)

Comment (scottrippey) [reply package.json:48]: After investigating the git history, looks like this lib was used for an old Markdown component.  
So yes, safe to delete now!

File: package.json (hunk 2 above)
Comment (scottrippey) [reply package.json:54]: Looks like this lib was used for an old Deck component.
So safe to delete too!

**Pull request merged**